basedtyping/__init__.py

```diff
@@ -68,7 +84,7 @@ class NotEnoughTypeParametersError(ReifiedGenericError):
     """Raised when type parameters are passed to a ``ReifiedGeneric`` with an incorrect number of type parameters:
 
     for example:
-    >>> class Foo(ReifiedGeneric[tuple[T, U]]):
+    >>> class Foo(ReifiedGeneric[Tuple[T, U]]):
```

Comment:
if youre changing mypy to target 3.10 why change these? shouldnt mypy be set to target an ancient python version instead

Reply:
The tests can be 3.10, but the arc needs to be 3.7.

Reply:
what that means

Reply:
tests are 3.7 now